.devcontainer/requirements.txt

```diff
@@ -0,0 +1,47 @@
+pillow>=3.4,<=5.1.1
```

Comment:
This looks like we now have to maintain two requirements files? Or introduces the possibility that this one becomes out-of-sync with the "real" one.

Reply:
Thanks, Dan. Yes, two requirements files, and yes, manual intervention is required to keep them in sync.  My sense is that the package requirements haven't been changing very frequently.  Specifying the current ones early in the Docker build makes rebuilding the image a lot quicker after a code change.  It's a tradeoff.

Reply:
You should feel free to merge this!